charts/fybrik/values.yaml

```diff
@@ -13,9 +13,10 @@ fullnameOverride: ""
 
 clusterScoped: true
 
+# If we work with "clusterScope:false", the namespace must already exist
 modulesNamespace: "fybrik-blueprints"
 
-# Set if need to only watch for FybrikApplication from a specific namespace
+# Set if needs to watch for FybrikApplication from a specific namespace only, or when we work with "clusterScope:false"
```

Comment:
No need to change the comment, clusterScope:false is used for watching FybrikApplications from a single namespace

Reply:
no, it works for clusterScope:true too, if it is defined, only a single namespace is watching.